gprofiler/main.py

```diff
@@ -142,18 +142,19 @@ def __init__(
         # 2. accessible only by us.
         # the latter can be root only. the former can not. we should do this separation so we don't expose
         # files unnecessarily.
+        if self._enrichment_options.container_names:
+            container_names_client: Optional[ContainerNamesClient] = ContainerNamesClient()
+        else:
+            container_names_client = None
```

Comment:
Can be a short `ContainerNamesClient() if ... else None` in the call to the ProfilerState c'tor